platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.kt

```diff
@@ -5,6 +5,7 @@ import android.os.Handler
 import android.os.Looper
 import androidx.annotation.IntDef
 import androidx.annotation.Keep
+import androidx.compose.runtime.getValue
```

Comment:
Why was this import required? It's a bit weird that we have to add a new dependency just for for that?

Reply:
The Kotlin rewrite changed the property name from 'GetID' to 'GetId' and to maintain forward and backward compatibility I have introduced a a delegated property `val ID: Long by this::id` which requires this import.  If a decision is made to target backward compatibility only then this can obviously be dropped

Reply:
@boldtrn I managed to build and run even if I commented out this line.

Reply:
>  'GetID' to 'GetId' 

Since this was changed in a major release it should be fine to rename this? But maybe we should add it to the Changelog to make it clear to everyone that it was changed? IMHO backwards compatibility for major releases can break.

Thanks for verifying @ovivoda.

Reply:
As you point out, it was changed in a major release but it still seems a shame to break existing code when the intention of a rewrite from Java to Kotlin was not to introduce any new functionality.  I am just a user of the library, not my call on how best to proceed, I'm happy to close the PR and fix it in [flutter-maplibre-gl](https://github.com/maplibre/flutter-maplibre-gl). Thanks for the amazing library!